crates/ironrdp-pdu/src/gcc/network_data.rs

```diff
@@ -62,7 +62,7 @@ impl ChannelName {
     ///
     /// # Panics
     ///
-    /// Panics if input is not null-terminated.
+    /// If input is not null-terminated.
```

Comment:
note: The way it originally was is more common.

See std library documentation :

<img width="1086" height="712" alt="Image" src="https://github.com/user-attachments/assets/909b6785-6c58-45bc-a9b0-581931c0f340" />